components/Navbar/index.tsx

```diff
@@ -27,7 +27,7 @@ function Navbar({
 }: NavbarProps) {
   const notification = showNotification ? (
     <>
-      Phase 2 Incentivized Testnet starts May 10th @ 12:00 PDT!
+      Phase 2 Incentivized Testnet started May 10th @ 12:00 PDT and will end November 15th!
```

Comment:
This is still hidden. We're gonna keep it hidden and were deploying the count down timer. We can show this after it ends.

```suggestion
      Phase 2 Incentivized Testnet ended November 15th @ 12:00 PDT
```

Reply:
Done